src/fuse.js

```diff
@@ -10,7 +10,10 @@ define([
     util
 ) {
     var FUSE_CONFIG = {
-        keys: ['label', 'name', 'absolutePath', 'hashtagPath'],
+        keys: ['displayPath'],
```

Comment:
Removing `label` because it doesn't make much sense to be able to search by the label (itext) when starting with #form or #case (i think)

`name === absolutePath` `displayPath = mug.form.useRichText ? mug.hashtagPath : mug.absolutePath`  which looks like what we want to do